pkg/framework/plugins/removeduplicates/removeduplicates_test.go

```diff
@@ -187,105 +189,111 @@ func TestFindDuplicatePods(t *testing.T) {
 		pods                    []*v1.Pod
 		nodes                   []*v1.Node
 		expectedEvictedPodCount uint
-		strategy                api.DeschedulerStrategy
+		excludeOwnerKinds       []string
+		nodefit                 bool
 	}{
 		{
 			description:             "Three pods in the `dev` Namespace, bound to same ReplicaSet. 1 should be evicted.",
 			pods:                    []*v1.Pod{p1, p2, p3},
 			nodes:                   []*v1.Node{node1, node2},
 			expectedEvictedPodCount: 1,
-			strategy:                api.DeschedulerStrategy{},
+			excludeOwnerKinds:       []string{},
```

Comment:
All `excludeOwnerKinds:       []string{},` lines can be dropped as `[]string{}` is the default value.